runtime/doc/channel.txt

```diff
@@ -696,6 +696,9 @@ See |job_setoptions()| and |ch_setoptions()|.
 "block_write": number	only for testing: pretend every other write to stdin
 			will block
 
+"env": dict		environment variables for new process
```

Comment:
Can we add a note for these 2 new options about the version of Vim which these are available?

Reply:
you can check with `:if v:version > 800 || v:version == 800 && has("patch902")`

Reply:
@chrisbra `:if has('patch-8.0.0902')` is better, I think.

@alanhamlett Unfortunately, such patch numbers are usually not documented.